osu.Game/Graphics/UserInterface/OsuTabControl.cs

```diff
@@ -59,23 +67,22 @@ public OsuTabControl()
         [BackgroundDependencyLoader]
         private void load(OsuColour colours)
         {
-            if (accentColour == default)
+            if (accentColour.Value == default)
                 AccentColour = colours.Blue;
         }
 
-        private Color4 accentColour;
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            accentColour.BindValueChanged(OnAccentColourChanged, true);
+        }
 
-        public Color4 AccentColour
+        protected virtual void OnAccentColourChanged(ValueChangedEvent<Color4> colour)
```

Comment:
Make `AccentColour` protected and bind each usage rather than making a virtual event. the same as done everywhere else in the game.